nmrfx-analyst/src/main/java/org/nmrfx/processor/datasets/vendor/NMRDataUtil.java

```diff
@@ -344,12 +347,27 @@ public static List<Path> findProcessedFiles(Path path) throws IOException {
                         -> {
                     String name = p.getFileName().toString();
                     return name.endsWith(".nv") || name.endsWith(".ucsf")
-                            || BrukerData.isProcessedFile(name);
+                            || BrukerData.isProcessedFile(name) || name.endsWith(RS2DData.PROC_DIR);
```

Comment:
Wouldn't it be better to do a recursive file search instead of this + the modification loop?

Reply:
Used Files.walkFileTree instead and removed modification loop.